src/openforms/authentication/contrib/tests/saml_utils.py

```diff
@@ -18,11 +19,8 @@ def create_test_artifact(service_entity_id: str = "") -> str:
 
 
 def get_artifact_response(filepath: str, context: Optional[dict] = None) -> bytes:
-    with open(filepath, "r") as template_source_file:
-        template = Template(template_source_file.read())
-
-    rendered = template.render(Context(context or {}))
-    return rendered.encode("utf-8")
+    template_source = Path(filepath).read_text()
+    return render_from_string(template_source, context or {}).encode("utf-8")
```

Comment:
it's not needed in this case, but it's also overkill to set up yet another backend for a test utility :)

The point of making this change is more to prevent people from copy-paste more `Template(...).render(Context(...))` code because they came across it, instead I want people to only find `render_from_string` and use that API.

So this is mostly a cosmetic change.